apps/mark-scan/frontend/src/pages/diagnostics/diagnostics_screen.tsx

```diff
@@ -112,10 +112,16 @@ export function DiagnosticsScreen({
               mostRecentPaperHandlerDiagnostic={
                 mostRecentPaperHandlerDiagnostic
               }
-              onStartPaperHandlerDiagnostic={() => {
-                startPaperHandlerDiagnosticMutation.mutate();
-                history.push('/paper-handler');
-              }}
+              paperHandlerSectionChildren={
+                <Button
+                  onPress={() => {
+                    startPaperHandlerDiagnosticMutation.mutate();
+                    history.push('/paper-handler');
+                  }}
+                >
+                  Test Printer/Scanner
+                </Button>
+              }
```

Comment:
Per @adghayes this is the preferred pattern for excluding the `Test` buttons from the readiness report (ie. we just omit the `Children` props in the readiness report)